documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -2516,25 +2516,47 @@ host    replication     <youruser>  ::1/128                 trust   // <3>
 For more information about network masks, see link:https://www.postgresql.org/docs/current/static/datatype-net-types.html[the PostgreSQL documentation].
 ====
 
-ifdef::community[]
 [[supported-postgresql-topologies]]
 === Supported PostgreSQL topologies
 
 The PostgreSQL connector can be used with a standalone PostgreSQL server or with a cluster of PostgreSQL servers.
 
-PostgreSQL (for all versions <= 16) supports logical replication slots on only `primary` servers. This means that a replica in a PostgreSQL cluster cannot be configured for logical replication, and consequently that the {prodname} PostgreSQL connector can connect and communicate with only the primary server. Should this server fail, the connector stops. When the cluster is repaired, if the original primary server is once again promoted to `primary`, you can restart the connector. However, if a different PostgreSQL server _with the plug-in and proper configuration_ is promoted to `primary`, you must change the connector configuration to point to the new `primary` server and then you can restart the connector.
+==== PostgreSQL 15 or earlier clusters
 
-When using PostgreSQL 16 or newer, you can set up logical replication slots also on replica servers,
-and {prodname} does not need to connect to your primary server for ingesting change events.
-In comparison to connecting to the primary,
-connecting to a replica will yield slightly increased latencies, though.
+When you deploy {prodname} in environments that run PostgreSQL 15 or earlier, you can configure logical replication slots only on the primary server in the cluster.
+You cannot configure logical replication on replica servers in the cluster.
+Consequently, the {prodname} PostgreSQL connector can connect and communicate only with the primary server.
+If the primary server fails, the connector stops.
 
-When using PostgreSQL 17 or newer, you can set up logical replication slots on primary serves which are enabled for failover.
-The state of such a failover slot can automatically be propagated to one or more replica servers,
-allowing {prodname} to continue to ingest changes from a replica promoted to primary after a failure,
-without missing any events.
+Following a failure, after the cluster is repaired, you can once again promote the original primary server to `primary`, and then restart the connector.
+Alternately, you might promote a different PostgreSQL server to `primary`.
```

Comment:
I wonder if it would be helpful to consider including a procedure like we do for **Upgrading PostgreSQL** also to outline the expected steps when you fail over from a primary to a replica for PostgreSQL 15 or earlier. It's often one of the things we need to explain.